OTAcceleratorCore/OTMultiPartyCommunicator.m

```diff
@@ -385,10 +394,27 @@ - (void)session:(OTSession *)session streamDestroyed:(OTStream *)stream {
     }
 }
 
+- (void)session:(OTSession*) session
+connectionCreated:(OTConnection*) connection {
+    _internalConnectionCount++;
+    
+    //check creationtime of the connections
+    [self compareConnectionTimeWithConnection: connection];
+}
+
+- (void)session:(OTSession*) session
+connectionDestroyed:(OTConnection*) connection {
+    _internalConnectionCount--;
+    
+    //check creationtime of the connections
+    [self compareConnectionTimeWithConnection: connection];
+}
```

Comment:
just to clarify, a new stream doesn't mean a new connection. Imagine I am connected to the session but I am not publishing --> a new connection will be created but not a new stream